frontend/src/features/workspace/components/AdminFeedbackModal/AdminFeedbackModal.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
+import { Modal, ModalContent, UseDisclosureReturn } from '@chakra-ui/react'
+
+import { AdminFeedbackModalContent } from './AdminFeedbackModalContent'
+
+export type AdminFeedbackModalProps = {
+  isOpen: UseDisclosureReturn['isOpen']
+  onClose: UseDisclosureReturn['onClose']
+  // we use a customise onOpen method unrelated to useDisclosure
+  onOpen: () => void
+}
+
+export const AdminFeedbackModal = ({
+  isOpen,
+  onClose,
+  onOpen,
+}: AdminFeedbackModalProps): JSX.Element => {
+  useEffect(() => {
+    if (isOpen) onOpen()
+  }, [onOpen, isOpen])
+
+  return (
+    <Modal blockScrollOnMount={false} isOpen={isOpen} onClose={onClose}>
```

Comment:
Maybe we can just use a sticky flex box (`"position": "fixed"`) instead of a modal for this? Can refer to @mayying's issue feedback button [here](https://github.com/opengovsg/FormSG/pull/6452/files#diff-9108ba10b9be696d2769f8f52722ffd38e377a30071456038317a0923b46ee2a).

Reply:
it might not be scrollable because the stuff have not been rendered yet? not sure

Reply:
Thanks @LinHuiqing, yeah after speaking with Ken, I think there might not be a need to use a modal at all, and maybe just wrap it in a Flexbox and conditionally render it. Will make the modification!

Reply:
[177d954](https://github.com/opengovsg/FormSG/pull/6465/commits/177d954b87dc820ae14de43fc93dfc686775be09)

have updated it! Quite a big refactor hahah do take a look ty!